bazel/cel-cpp.patch

```diff
@@ -0,0 +1,1360 @@
+diff --git a/base/BUILD b/base/BUILD
+index 2ddce9f..f6c1d58 100644
+--- a/base/BUILD
++++ b/base/BUILD
+@@ -13,7 +13,7 @@ cc_library(
+         "statusor.h",
+         "statusor_internals.h",
+     ],
+-    copts = ["-std=c++14"],
```

Comment:
same here.

Reply:
Need to verify that .bazelrc cxxopts propagate to cell-cpp dependency. But eliminating the duplicates would be fantastic.

We expect to also pick up c++17, the msvc / clang-cl syntax is just slightly different, as illustrated in bazel/envoy_internal.bzl

Reply:
Unfortunately the flag doesn't propagate to cel-cpp dependency.